test/scripts/addFakeAuthentication.sh

```diff
@@ -0,0 +1 @@
+docker-compose exec -T db mysql -u root < ./test/scripts/addFakeAuthentication.sql
```

Comment:
@m273d15 Execute this script or line after the db container is ready.

Reply:
What does it do?